build.sbt

```diff
@@ -2,7 +2,7 @@ import scala.collection.mutable
 import scala.sys.process._
 import Tests._
 
-def localSnapshotVersion = "0.10.7-SNAPSHOT"
+def localSnapshotVersion = "0.1.7-SNAPSHOT"
```

Comment:
unintended?

Reply:
:man_facepalming: I was testing some things out and totally forgot about it.